neutron/assetlist.json

```diff
@@ -1242,6 +1242,42 @@
           }
         }
       ]
+    },
+    {
+      "description": "Mars Protocol is a cross-collateralized Money Market Protocol on Neutron and Osmosis.",
+      "denom_units": [
+        {
+          "denom": "factory/neutron1ndu2wvkrxtane8se2tr48gv7nsm46y5gcqjhux/MARS",
+          "exponent": 0,
+          "aliases": ["umars"]
+        },
+        {
+          "denom": "MARS",
+          "exponent": 6
+        }
+      ],
+      "base": "factory/neutron1ndu2wvkrxtane8se2tr48gv7nsm46y5gcqjhux/MARS",
+      "name": "Mars Protocol token",
```

Comment:
Because Mars is being replaced, please add a "traces" to the original Mars asset using type: "legacy-mintage", Provider: "Mars Protocol". This will signal that it has canonically been replaced.

Reply:
actually it would be the mars/assetlist.json::umars asset that needs the legacy-mintage, because that is the one become legacy. This asset should not have traces because it's the new 'canonical' asset.